docs/installation/installing-with-npm.md

```diff
@@ -1,303 +1,196 @@
-# Installing GOV.UK Frontend with node package manager (NPM)
+# Install GOV.UK Frontend using Node.js package manager (npm)
 
 ## Requirements
 
-To use GOV.UK Frontend with NPM you must:
+1. Install [Node.js](https://nodejs.org/en/).
 
-1. Install the long-term support (LTS) version of
-   [Node.js](https://nodejs.org/en/), which includes NPM. The minimum version of
-   Node required is 4.2.0.
+    If you have not already installed version 4.2.0 or later of Node.js, install the latest Long Term Support (LTS) version.
 
-   (We recommend using [`nvm`](https://github.com/creationix/nvm) for managing
-   versions of Node.)
+2. `cd` to the root of your project and check if you have a [`package.json` file](https://docs.npmjs.com/files/package.json). If you do not have the file, create it by running:
 
-2. Create a [package.json file](https://docs.npmjs.com/files/package.json) if
-   you don’t already have one. You can create a default `package.json` file by
-   running `npm init` from the root of your application.
+    ```
+    npm init -y
```

Comment:
Can you explain the decision to say `-y`, this will mean they accept a code licence that they may not agree with.

Reply:
If I understand right, this was to avoid people being presented with all the package.json questions that aren't very relevant for them and potentially hard to answer/explain. @36degrees - have I got that right?

Reply:
People should make the decision what licence their code is in as it has legal implications and can impact how the code is shared.

Reply:
Yeah, I hadn't twigged that one of the questions was about the license – that probably shouldn't be something that gets accepted automatically. Good spot, @nickcolley. I think we probably need to revert this and provide guidance around how to answer the questions that they're asked (which I guess will include providing _something_ to help people understand how to answer the licensing question… 😬)

Do package.json files require a license field? Is it something users can skip without specifying?

Reply:
We might want to just say `npm init` and link to the official guidance: https://docs.npmjs.com/creating-a-package-json-file